quickstart/armv7a9-zynq7000-zturn.md

```diff
@@ -0,0 +1,245 @@
+  shine blue.
+
+- Now you should verify, what USB device on your host-pc is connected with the `UART` (console). To check that run:
+
+  ```plaintext
+  ls -l /dev/serial/by-id
+  ```
+
+  <img src="_images/zynq7000-zturn-ls.png" width="700px">
+
+  If your output is like in the screenshot above, the console (`UART` in the evaluation board) is on the `USB0` port.
+
+- When the board is connected to your host-pc, open serial port in terminal using picocom and type the console port
+  (in this case USB0)
+
+  ```plaintext
+  picocom -b 115200 --imap lfcrlf /dev/ttyUSB0
+  ```
+
+- You should see such output:
+  
+  <img src="_images/zynq7000-zturn-picocom.png" width="700px">
+
+  <details>
+  <summary>How to get picocom (Ubuntu 20.04)</summary>
+
+  ```plaintext
+  sudo apt-get update && \
+  sudo apt-get install picocom
+  ```
+
+  To use picocom without sudo privileges run this command and then restart:
+
+  ```plaintext
+  sudo usermod -a -G dialout <yourname>
+  ```
+
+  </details>
+  </br>
+
+You can leave the terminal with the serial port open, and follow the next steps.
+
+## Flashing the Phoenix-RTOS system image
+
+At first, before any flashing, you need to enter Phoenix-RTOS loader (plo).
+
+### Entering Phoenix-RTOS loader (plo)
+
+Press RESET button (`K2`) to restart the chip.
+
+If flash memory doesn't contain Phoenix-RTOS system image, booting process will stop at plo level, you should see:
+
+<img src="_images/zynq7000-zturn-magic.png" width="700px">
+
+Phoenix-RTOS loader tried to find an image in flash, which was unsuccessful. That's why there is an error message.
+
+Otherwise, in order to get into plo you need to press any key within 500ms. If you don't do that, plo will load system
+and start psh, but we cannot flash from there. Output when you enter psh:
+
+<img src="_images/zynq7000-zturn-psh.png" width="700px">
+
+Restart the chip with RESTART button `K2` and try again. Output of successful entry to plo:
+
+<img src="_images/zynq7000-zturn-plo.png" width="700px">
+
+If you want to flash the system image please follow the next steps.
+
+### Copying image to flash memory using PHFS (phoenixd)
+
+To flash the disk image, first, you need to verify on which port plo USB device has appeared. You can check that using
+`ls` as follow:
+
+```plaintext
+ls -l /dev/serial/by-id
+```
+
+<img src="_images/zynq7000-ls-2.png" width="700px">
+
+To share disk image to the bootloader, `phoenixd` has to be launched with the following arguments (choose suitable
+ttyACMx device, in this case, ttyACM0):
+
+```plaintext
+cd _boot/armv7a9-zynq7000-zturn
+```
+
+```plaintext
+sudo ./phoenixd -p /dev/ttyACM0 -b 115200 -s .
+```
+
+<img src="_images/zynq7000-zturn-phoenixd.png" width="700px">
+
+Before flashing, good practise is to erase older file system on flash memory (this is done to avoid errors).
+
+If you encountered some problems during this step please see
+ [common problems](armv7a9-zynq7000.md#common-problems-on-zynq7000-boards).
```

Comment:
Rather this way, you are not refferring to erasing step.

```suggestion
If you encountered some problems during this step please see
 [common problems](armv7a9-zynq7000.md#common-problems-on-zynq7000-boards).

Before flashing, good practise is to erase older file system on flash memory (this is done to avoid errors).
```